website/docs/docs/cloud/about-cloud-develop-defer.md

```diff
@@ -37,10 +37,13 @@ When using defer, it compares artifacts from the most recent successful producti
 
 To enable defer in the dbt Cloud IDE, toggle the **Defer to production** button on the command bar. Once enabled, dbt Cloud will:
```

Comment:
the spacing is little off, might need to indent the paragraphs

![Screenshot 2025-01-31 at 11 24 32](https://github.com/user-attachments/assets/79c9ced8-18c6-45ea-92d3-6d1998ebe80b)